src/pbt/v2/deployment/pipeline.py

```diff
@@ -244,7 +249,8 @@ def _get_package_name(self):
 
     def mvn_build(self):
         mvn = "mvn"
-        command = [mvn, "package", "-DskipTests"] if not self._are_tests_enabled else [mvn, "package"]
+        command = [mvn, "package", "-DskipTests"] if not self._are_tests_enabled else [mvn, "package",
+                                                                                       "-Dfabric=default"]
```

Comment:
Do we not need `"-Dfabric=default"` in the skip-test link?

Reply:
That is when tests are running.